src/rust/rust-proto/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "rust-proto"
 version = "0.2.10"
-authors = ["Insanitybit <insanitybit@gmail.com>"]
+authors = ["Grapl <colin@graplsecurity.com>"]
```

Comment:
This is also a weird change to have in this PR.

Reply:
reverted

Reply:
Reverted